package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@toyota-research-institute/lakefront",
   "description": "React UI Components",
-  "version": "0.4.4",
+  "version": "0.4.4-button.3",
```

Comment:
Does this need to change before it gets merged?

Reply:
The merge to master github action should take care of the version merging when this gets merged to main.